### PR TITLE
docs: add k8s slack channel

### PR DESCRIPTION
**Pull request by cdrage**
docs: add k8s slack channel

### What does this PR do?

Adds the Kubernetes slack channel to the site / README as well as Podman
Desktop help page

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,10 @@ For all bug and feature requests please [file a GitHub issue](https://github.com
 
 Discussions are done using [Github Discussions](https://github.com/containers/podman-desktop/discussions/).
 
+Join #podman-desktop on the [Kubernetes Slack](https://communityinviter.com/apps/kubernetes/community).
```
Comment (benoitf) [inline README.md]: We should probably have the direct link to the channel as well (if people are already on the instance) https://app.slack.com/client/T09NY5SBT/C04A0L7LUFM

Here it looks like we're displaying the 3rd party provider url but instead maybe we need to use the alias from kubernetes/k8s domain http://slack.k8s.io/

**Comment by cdrage** [comment]
@benoitf Updated!

**Review by benoitf — APPROVED** [review]
LGTM we can always iterate later
👍

**Comment by benoitf** [comment]
I didn't see you updated the PR so I approved lately

**Pull request merged**